### PR TITLE
Updated average.js to get the right output

### PR DESCRIPTION
**Pull request by tago893**

### CODE REVIEW

File: average.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  const noNaNnumber=numbers.filter(x => !Number.isNaN(x))
+  return noNaNnumber.reduce((p, c)=> p + c, 0) / noNaNumber.length;
 }
 
 module.exports = {average};
```
Comment (numocityadmin) [inline average.js:7]: Try a new test: The tests in `test/average.test.js` express the functionality of the `average` function.
The functionality accepts a sequence of readings from an IoT sensor. The sensor sends `NaN` when something goes wrong in measurement, which we have ignored.
Problem: If the function sends a lot of NaNs and very few readings, something is wrong. However, the `average` function ignores all the NaNs and still computes the average, as if nothing went wrong.
We need to add functionality to the average function to return `NaN`, when most of the input-numbers are `NaN`.
You need to:

- Reason out what 'most of the numbers' means: E.g., determine the number of NaNs in the input which would indicate that the sensor is unreliable. Mention any references you used to arrive at this logic
- Add a test that supplies such inputs and expects the output to be NaN
- Add code that satisfies this functionality, while not breaking existing functionality.

**Review by numocityadmin — COMMENTED** [review]
Great start!
See comment for adding a new functionality.
Give it a shot :)

File: average.js (hunk 1 above)
Comment (tago893) [reply average.js:7]: @numocityadmin Sir, I thought only in the feedback pull request you will reply so I deleted this branch  and after some time somehow I was able to get the feedback pull request open by GitHub classroom bot, can you please review the code in the feedback channel sir, or should I just restore this branch your wish sir which pull request you want me to open I will open it and the other one I will close it sir

Comment (numocityadmin) [reply average.js:7]: I see that you have written the code to pass the test, but I cannot see the code to report the average of NaN if too many numbers in the input are NaN (that is the functionality I requested to add in the comment above).
Please point me to the code which implements the additional functionality.

**Pull request closed without merging**